sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminClient.java

```diff
@@ -0,0 +1,114 @@
+package com.azure.ai.personalizer;
+
+import com.azure.ai.personalizer.implementation.PersonalizerClientV1Preview3Impl;
+import com.azure.ai.personalizer.implementation.PersonalizerClientV1Preview3ImplBuilder;
+import com.azure.ai.personalizer.implementation.models.*;
```

Comment:
[AvoidStarImport](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml#L63)